### ISSUE TITLE
Found the "U.S. Files" with state/county level data in Daily Report 03-23-2020

### ISSUE THREAD

**Issue by ei-JoanneT** [CSSEGISandData/COVID-19#1340]
As you might remember in #1250, there will be an update of Global file and U.S. file

After the U.S. file missing for too long, I found the 03-23-2020 is actually the U.S. file they suggested in #1250 with all state and county level data in U.S.

Should I start building something depending on that structure or I should cross my fingers for the old files to get corrected before 8 A.M. tomorrow...

**Comment by cortical-iv** [comment]
I don't understand  -- why make backwards incompatible changes to a database that literally hundreds of people are using for a public health function, right in the middle of a pandemic? How many coder-hours of time will now be wasted figuring this out?

**Comment by jtene001** [comment]
I agree. Thousands of people were using and relying on this database - and now you have a completely different structure. 

What is the reason for not including recovered? Before, you could calculate active cases. Now- nope.

**Comment by ei-JoanneT** [comment]
> I don't understand -- why make backwards incompatible changes to a database that literally hundreds of people are using for a public health function, right in the middle of a pandemic? How many coder-hours of time will now be wasted figuring this out?

It's been a respectful task of putting everything together for us in the first place, so I totally understand all changes if it could ever make their life easier. But if they could tell us ahead of time when they are going to make changes and where the files would be for future changes it will be great

**Comment by GitRobGit** [comment]
What principle of Github was violated by changing the structure midstream? Seems there would have been at least an informational note that this occurred, updated Readme, etc. Rather than having hundreds or thousands relying on this suddenly needing to recode. -Newbie

**Comment by Bmr55** [comment]
So, there will no longer be US state level time series data? Will we have to compile that ourselves through the daily reports?

**Comment by johnrolfeellis** [comment]
> Should I start building something depending on that structure or I should cross my fingers for the old files to get corrected before 8 A.M. tomorrow...

I'm also awaiting clarification on this.

**Comment by aliencraft1512** [comment]
There are certain inconsistencies in the new formatted data.(e.g. France statistics are attributed to French Polynesia instead.)

**Comment by msgoussi** [comment]
if you continue changing the name of the files, it will affect our dashboard and report. 
why the there is a such a file called "recovered-globally", since the recovered file is not updated.

**Comment by ei-JoanneT** [comment]
> if you continue changing the name of the files, it will affect our dashboard and report.
> why the there is a such a file called "recovered-globally", since the recovered file is not updated.

There is no "recovered_global" in the time-series folder, at least not there the last time I check
